### PR TITLE
fix(symfony) :  symfony security-http fixed

### PR DESCRIPTION
**Pull request by Drago37**
### FIXED

* symfony security-http fixed

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [x] 24.10

<h2> How this pull request can be tested ? </h2>

CI tests

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon-awie/composer.lock [hunk 1]
```diff
@@ -450,7 +450,7 @@
                 "psr/container": "1.1.1",
                 "sensio/framework-extra-bundle": "^6.2",
                 "smarty-gettext/smarty-gettext": "^1.6",
-                "smarty/smarty": "^v4.3",
+                "smarty/smarty": "^v4.5",
```
Comment (sc979) [inline centreon-awie/composer.lock:453]: was smarty upgrade intended ?

Comment (Drago37) [reply centreon-awie/composer.lock:453]: It's ok for you ? @sc979

Comment (sc979) [reply centreon-awie/composer.lock:453]: yes of course it is 👍

Comment (Drago37) [reply centreon-awie/composer.lock:453]: Ok I resolve thanks :)

**Pull request merged**